src/app/templates/partials/downloads.ftl

```diff
@@ -1 +1,61 @@
-<h2>Downloads</h2>
\ No newline at end of file
+<#-- @ftlvariable name="report" type="org.vaccineimpact.orderlyweb.models.ReportVersionDetails" -->
+<#-- @ftlvariable name="artefacts" type="List<org.vaccineimpact.orderlyweb.controllers.web.ArtefactViewModel>" -->
+<#-- @ftlvariable name="dataLinks" type="List<org.vaccineimpact.orderlyweb.controllers.web.InputDataModel>" -->
+<#-- @ftlvariable name="resources" type="List<org.vaccineimpact.orderlyweb.controllers.web.DownloadableFileViewModel>" -->
+<#-- @ftlvariable name="zipFile" type="org.vaccineimpact.orderlyweb.controllers.web.DownloadableFileViewModel" -->
+
+<#import "../macros/file-download-link.ftl" as file_download_link>
+
+<#include "report-title.ftl">
+
+<div id="artefacts">
+    <#list artefacts as artefact>
+        <div class="mb-2 card">
+            <div class="card-header">${artefact.artefact.description}</div>
+            <div class="card-body">
+                <#if artefact.inlineArtefactFigure??>
+                    <img src="${artefact.inlineArtefactFigure}" class="border border-dark p-3 col-12 col-lg-8">
```

Comment:
The image and the first link are a bit close, in the reportle we have an `mt-2` class on the first link, but at `mb-2` on the `img` tag would also do the trick.